TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -298,9 +298,18 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 		_mediaBrowserView: null,
 
 		_beforeMediaBrowserIsShown: function () {
-			var that = this;
+			var that = this,
+				value = null;
+
+			try {
+				value = JSON.parse(this.get('value'));
+			} catch (exception) {}
```

Comment:
hmm in that case I'd actually prefer an exception instead of it silently being swallowed.. I'd go with s.th. like instead, but guess that's a matter of preference.. anyway I'd imagine a lot of other stuff breaking before with an invalid value anyway

    value = this.get('value') ? JSON.parse(this.get('value')) : null;